src/redux/epics/event.epics.js

```diff
@@ -18,15 +22,42 @@ export const getEventsEpic = (action$) =>
   action$.ofType(ACTION_TYPES.GET_EVENTS)
     .mergeMap(action =>
       Observable.ajax({
-        url: `${BASE_ENDPOINT}${action.payload}`,
+        url: `${BASE_ENDPOINT}${CATEGORY_METHOD_PATH}${API_KEY_PATH}${CATEGORY_QUERY_PATH}${action.payload}`,
         crossDomain: true
       })
       .map(({ response }) => ({
         type: ACTION_TYPES.SET_EVENTS,
         payload: response,
       }))
       .catch(error => Observable.of({
-        type: ACTION_TYPES.SET_EVENT_INVALIDATE,
+        type: ACTION_TYPES.SET_EVENTS_INVALIDATE,
         payload: error
       })) 
     );
+
+// SECTION: Single event
```

Comment:
Would it make sense to split these into two epics? It would make the code easier to understand, and it would match the structure of the reducers.